x-pack/plugins/translations/translations/zh-CN.json

```diff
@@ -463,9 +463,9 @@
     "common.ui.savedObjects.confirmModal.overwriteTitle": "覆盖“{name}”？",
     "common.ui.savedObjects.confirmModal.saveDuplicateButtonLabel": "保存“{name}”",
     "common.ui.savedObjects.confirmModal.saveDuplicateConfirmationMessage": "具有标题 “{title}” 的 “{name}” 已存在。是否确定要保存？",
-    "common.ui.savedObjects.howToSaveAsNewDescription": "在 Kibana 的以前版本中，更改 {savedObjectName} 的名称将创建具有新名称的副本。使用“另存为新的 {savedObjectName}” 复选框可立即达到此目的。",
+    "timelion.savedObjects.howToSaveAsNewDescription": "在 Kibana 的以前版本中，更改 {savedObjectName} 的名称将创建具有新名称的副本。使用“另存为新的 {savedObjectName}” 复选框可立即达到此目的。",
     "common.ui.savedObjects.overwriteRejectedDescription": "已拒绝覆盖确认",
-    "common.ui.savedObjects.saveAsNewLabel": "另存为新的 {savedObjectName}",
+    "timelion.savedObjects.saveAsNewLabel": "另存为新的 {savedObjectName}",
```

Comment:
Don't we need to move it manually to a group of timelion translations?
Because I did it in my PRs ... 😩

Reply:
You're right, thanks for that hint. I've improved that! you did it right 🎁